src/combobox/ComboBox.ts

```diff
@@ -394,7 +396,7 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 			v('button', {
 				'aria-controls': menuId,
 				bind: this,
-				classes: this.classes(css.arrow),
+				classes: this.classes(css.trigger, iconCss.icon, iconCss.chevronDown),
```

Comment:
I think this is a pretty common way of doing things. If we left `iconCss` out then the icons would not be themeable so I don't think that is an issue. I believe we made a decision to use an icon font to provide icons as part of the original issue and this way we have the css-modules providing the icon type-ahead etc... It's quick and easy for a user to see what icons are available to them.